sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/types/PhysicalDataType.scala

```diff
@@ -326,6 +327,17 @@ case class PhysicalStructType(fields: Array[StructField]) extends PhysicalDataTy
   }
 }
 
+class PhysicalVariantType extends PhysicalDataType {
```

Comment:
can we add a comment for this?

Reply:
I don't feel there is anything special and worth a comment in this class. All `Physical*Type` classes in this file don't have a comment.